jsonnet/openshift-state-metrics.libsonnet

```diff
@@ -1,4 +1,20 @@
 function(params) {
+  // override_memory sets the memory request for a container to the given value.
```

Comment:
I am just starting with jsonnet. If there's a better way to accomplish this please make a suggestion :)

Reply:
you should be able to modify the limit using `$._config.baseMemory`:

https://github.com/openshift/openshift-state-metrics/blob/101429149266e2c86a41b82baf47a033c9a93b02/jsonnet/openshift-state-metrics.libsonnet#L12-L15

Reply:
Good point, should be there now